### PR TITLE
add analyst guide and add https

### PR DESCRIPTION
**Pull request by xjackk**
* https://federal-spending-transparency.atlassian.net/browse/DA-5380

Really quick check on the `resources` section.

### CODE REVIEW

**Comment by xjackk** [comment]
EDIT:

I added https://github.com/fedspendingtransparency/datalab/pull/278/commits/67ad938cdc387577de864a30c8ed14c870e59d2f to fix the `resources-row.jsx`. using Gatsby's `Link` is for internal links between pages as it appends `page` to the url. These were all broken. Using `a` is the exact same thing and is used for external links - linking to our API and the pdf file.

**Comment by anjenkin** [comment]
Looks good!

File: src/page-sections/home/resources-row/resources-row.jsx [hunk 1]
```diff
@@ -53,10 +52,10 @@ const ResourcesRow = () => {
             key={index}
             className={resourcesRowStyles.tileRow}
           >
-            <Link
+            <a
               key={'landing-row__tile_' + index}
               className={resourcesRowStyles.tile}
-              to={resource.href}
+              href={resource.href}
```
Comment (anjenkin) [inline src/page-sections/home/resources-row/resources-row.jsx]: Does the a tag work correctly for both the internal link and the external link?

Comment (anjenkin) [reply src/page-sections/home/resources-row/resources-row.jsx]: Just verified this works locally.

**Pull request merged**